### PR TITLE
add ability to add headers in response for mock

### PR DESCRIPTION
**Pull request by APshenkin**

### CODE REVIEW

File: src/rules/handlers.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@ import https = require('https');
 import express = require("express");
 import { OngoingRequest } from "../types";
 import { RequestHandler } from "./mock-rule-types";
+import { OutgoingHttpHeaders } from "http";
```
Comment (pimterry) [inline src/rules/handlers.ts]: `http` is actually already imported above - I think if you just replace `OutgoingHttpHeaders` below with `http.OutgoingHttpHeaders` you can drop this entirely. Or you could change the above `http` import to use destructuring too, up to you.

File: src/rules/handlers.ts [hunk 2]
```diff
@@ -44,9 +46,9 @@ export function buildHandler
 }
 
 const handlerBuilders: { [T in HandlerType]: HandlerBuilder<HandlerDataLookup[T]> } = {
-    simple: ({ data, status }: SimpleHandlerData): RequestHandler => {
+    simple: ({ data, status, headers }: SimpleHandlerData): RequestHandler => {
         let responder = _.assign(async function(request: OngoingRequest, response: express.Response) {
-            response.writeHead(status);
+            response.writeHead(status, headers);
             response.end(data || "");
         }, { explain: () => `respond with status ${status}` + (data ? ` and body "${data}"` : "") });
```
Comment (pimterry) [inline src/rules/handlers.ts]: The headers aren't included in the explanation for this rule. Can you expand this, so if headers are set it includes them here in some sensible format, as it does with the body?

**Comment by pimterry** [comment]
Hi, doesn't look there's been any updates on this. Anything I can do to help?

**Comment by APshenkin** [comment]
Sorry, I just noticed the messages. Looks like now everything should be ok.

**Comment by pimterry** [comment]
Good stuff, thanks! Merged.

**Pull request merged**